src/main/java/mekanism/client/gui/GuiChemicalCrystallizer.java

```diff
@@ -48,7 +49,7 @@
 
 	public Gas prevGas;
 
-	public ItemStack renderStack;
+	public @Nullable ItemStack renderStack;
```

Comment:
It is unlikely being a GUI class, and that would be the point of annotating it (and also why I mentioned Aidan might want to review how his classes do it)